### PR TITLE
feat(state): implement clearedValue

### LINKED ISSUE #696: [manager] Implement clearedValue
- field level
- form level

### PR DESCRIPTION
**Pull request by rvsia**
closes #696

### CODE REVIEW

File: packages/form-state-manager/src/tests/utils/use-subscription.test.js [hunk 1]
```diff
@@ -211,4 +211,77 @@ describe('useSubscription', () => {
...
+  });
+
+  it('#checkEmpty - edge cases', () => {
+    expect(checkEmpty(false)).toEqual(false);
+    expect(checkEmpty('A')).toEqual(false);
+    expect(checkEmpty(0)).toEqual(false);
+
+    expect(checkEmpty([])).toEqual(true);
+    expect(checkEmpty('')).toEqual(true);
+    expect(checkEmpty(undefined)).toEqual(true);
```
Comment (Hyperkid123) [inline packages/form-state-manager/src/tests/utils/use-subscription.test.js:355]: So the big question here is. How do we treat null and empty objects?

Comment (rvsia) [reply packages/form-state-manager/src/tests/utils/use-subscription.test.js:355]: It's the same as before, just copy pasted code with a new test to get it covered. :man_shrugging: :smile: If we don't want to introduce breaking changes, the question is then already answered. :smile:
